demo/index.html

```diff
@@ -27,12 +27,31 @@
     </script>
   </head>
   <body>
-    <h1>Parent Page</h1>
+    <div
+      style="display: flex; align-items: center; justify-content: space-around"
+    >
+      <h1>Demo</h1>
+      <div>
+        <form id="commands">
+          <input type="text" name="command" placeholder="Enter a command" />
```

Comment:
Should we also have another input to specify (optional) args?

Reply:
Added!